packages/node/src/BacktraceNodeRequestHandler.ts

```diff
@@ -0,0 +1,107 @@
+import {
+    BacktraceAttachment,
+    BacktraceReportSubmissionResult,
+    BacktraceRequestHandler,
+    BacktraceSubmissionResponse,
+    DEFAULT_TIMEOUT,
+} from '@backtrace/sdk-core';
+import { BacktraceData } from '@backtrace/sdk-core/src/model/data/BacktraceData';
+import axios from 'axios';
```

Comment:
Can we _not_ use `axios`? Let's keep the dependency list as short as possible.

Instead, let's use `http` and `https`, and on Node 18, we can use `fetch`.

Reply:
instead of supporting http/https and fetch on node 18 it's worth sticking to one implementation. In addition to that on the server side, we don't care much about the package size. we used http/https in our migration script and until now I'm not sure how we can handle specific events generated by them. http/https might introduce additional pain and won't give us too much. 

I agree we should use less packages than more, but is there a real benefit for not using axios if we use it everywhere?